openjdk-8.yaml

```diff
@@ -0,0 +1,222 @@
+package:
+  name: openjdk-8
+  version: 8u372-b07
```

Comment:
seems like this isn't parseable by `wolfictl`, but I'm not sure this is the version naming we want to use given this is based on icedtea? follow up to that is how the icedtea versions translate to the openjdk versions...

Reply:
@rawlingsj may have advice.

Reply:
This should probably be 8.372.07.